build.gradle

```diff
@@ -54,3 +55,27 @@ compileTestKotlin {
 task wrapper(type: Wrapper) {
     gradleVersion = '4.7'
 }
+
+detekt {
+    input = files("src/main/kotlin")
+
+    config = files("config/detekt/detekt.yml")
+
+    ignoreFailures = true // If`true` the build doesn't fail when the maxIssues count was reached.
```

Comment:
I'd leave this on. Currently there are 9 issues, the default for failure seems to be 10 so it passes. Someone in the future will have to address them at some point though :D

Reply:
So, do I set it to false?

Reply:
It defaults to false, I'd just remove it (https://arturbosch.github.io/detekt/groovydsl.html).

Reply:
I've already done like that.
I've also done the rebase of master, so you don't have merge conflicts.